CREDITS.md

```diff
@@ -12,10 +12,11 @@ https://github.com/pbering/SockerVS2017
 https://github.com/pbering/solrwin
 
 **Joost Meijles (Aviva Solutions)**: His contributions in GitHub have been a source of inspiration. He is also active in community discussions about using Docker with Sitecore.
-https://github.com/sitecoreops/sitecore-images
+https://github.com/avivasolutionsnl/sitecore-commerce-docker
 https://github.com/joostmeijles/sitecore-commerce-docker
```

Comment:
Please remove this https://github.com/joostmeijles/sitecore-commerce-docker link (its only a not-well maintained fork of https://github.com/avivasolutionsnl/sitecore-commerce-docker)